devops/azure-pipelines/discord-job-template.yml

```diff
@@ -0,0 +1,25 @@
+jobs:
+  - job: DiscordMessage
+    dependsOn: ${{ parameters.dependOn }}
+    variables: 
+    - group: Discord-Webhook-Info
+    - name: discord_id
+      value: $[variables.discordid]
+    - name: discord_key
+      value: $[variables.discordkey]
+    steps: 
+    - task: ado-discord-webhook@1
+      inputs:
+        channelId: $(discord_id)
+        webhookKey: $(discord_key)
+        name: 'Team2-P3'
+        messageType: 'content'
+        content: |
+          **Team 2 Project 3 Frontend**: $(Build.SourceVersionMessage)
```

Comment:
cf. https://github.com/1058-Sophia-Gavrila-Forge/Forge-Backend/pull/2 on this file